.eslintrc

```diff
@@ -7,6 +7,7 @@
   },
   "extends": "airbnb-base",
   "rules": {
+    "no-restricted-globals": ["error", "Promise"],
```

Comment:
This will enable ESLint to output errors when we (mistakingly) use the global Promise constructor.